### PR TITLE
ZIOS-10799: Added loading of quotes from draft messages

### PR DESCRIPTION
**Pull request by nicorsm**
## What's new in this PR?

Quotes weren't loading correctly when saved inside the `DraftMessage` object after switching the conversation or at startup.

### Needs release with

- [x] https://github.com/wireapp/wire-ios-data-model/pull/601

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m [hunk 1]
```diff
@@ -194,6 +194,20 @@ - (void)viewDidLoad
     [self.view addGestureRecognizer:pinchImageGestureRecognizer];
     
     [self createMentionsResultsView];
+    
+}
+
+- (void)loadQuotedMessageIfNeeded {
+    
+    NSUUID *nonce = self.conversation.draftMessage.quote.nonce;
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m]: Can't the loading of the message be done in data model?

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m [hunk 2]
```diff
@@ -194,6 +194,20 @@ - (void)viewDidLoad
...
+    NSManagedObjectContext *context = self.conversation.managedObjectContext;
+    ZMMessage *quote = [ZMMessage fetchMessageWithNonce:nonce
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m]: May be this is better to do on the SE/DM side? Or as the extension. Like `self.conversation.draftMessage.quotedMessage?`. I think it's not the best to use the `NSManagedObjectContext` in UI directly.

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m]: Reading the data model code it looks like we are already loading the message.

**Comment by typfel** [comment]
@nicorsm let's bump data model and merge this :)

**Pull request merged**